docs/content/preview/architecture/docdb/persistence.md

```diff
@@ -102,7 +102,56 @@ The non-primary key columns correspond to subdocuments within the document. The
 
 We use a binary-comparable encoding to translate the value for each YCQL type to strings that go to the KV-Store.
 
-### Data expiration in YCQL
+## Packed Row format (Beta)
```

Comment:
do we need to call this Beta here?

Reply:
Yes, thats how most features are labelled right? Will switch it out of Beta once 2.16.1 ships.